code/game/machinery/newscaster.dm

```diff
@@ -887,6 +887,12 @@ GLOBAL_LIST_EMPTY(allCasters)
 	var/wantedBody
 	var/wantedPhoto
 	var/creationTime
+	var/datum/horoscope/horoscope = new /datum/horoscope
```

Comment:
can be simplified to just `new`
also, this should be done in `Initialize`, not the constructor